README.md

```diff
@@ -0,0 +1,44 @@
+# Welcome to PubDoc
+
+## TL;DR
+The goal of this repository is twofold:
+1. To make documentation better for the reader.
+2. To make _creating_ documentation a better experience for the writer.
+
+## How to Use
+This repository is currently very "bare bones." To run the tool and interface with Confluence,  
+you may need to contact an IT administrator to obtain permissions to access Confluence data.  
+
+Visit [this link](https://id.atlassian.com/manage-profile/security/api-tokens) to create your API token.  
+Next, create a `.env` file with the following contents:
+
+### `.env` File Example
+```env
+CONFLUENCE_USERNAME=<username>
+CONFLUENCE_API_KEY=<api key>
+CONFLUENCE_DOMAIN=<your company>.atlassian.net
+CONFLUENCE_SPACEID=<A space you don't care about littering in>
+```
+To get a spaceID of a space. Go to the space then click on "space settings" in the top left. Then run this in the browser to get the spaceID. `https://YOURURL.atlassian.net/wiki/rest/api/space/KEY`
```

Comment:
Wow, I didn't know about this and only found the API for getting all spaces in a domain. So I binary-searched all FlexGen spaces to find mine lol

Reply:
Yea a lot of confluence api usage seems to be more or less black magic. Found that in a confluence forum. 

It's the old api so idk how long it'll even work.